tensornetwork/contractors/opt_einsum_paths/path_contractors.py

```diff
@@ -17,9 +17,11 @@
 import functools
 import opt_einsum
 from tensornetwork.network import TensorNetwork
-from tensornetwork.network_operations import check_connected, get_all_edges
+# pylint: disable=line-too-long
+from tensornetwork.network_operations import check_connected, get_all_edges, get_subgraph_dangling
 # pylint: disable=line-too-long
 from tensornetwork.network_components import get_all_nondangling, contract_parallel
+    
```

Comment:
nit: probably no need for the newline here

Reply:
Removed